.travis.yml

```diff
@@ -4,7 +4,6 @@ sudo: false # faster builds
 jdk:
     - oraclejdk9
     - oraclejdk8
-    - openjdk7
```

Comment:
if we decide remove java7 support. should we have a separate pr instead of mixing in this pr?

Reply:
this pr should synchronize with the origin/master

Reply:
I will synchronize with the origin/master branch.

Reply:
> if we decide remove java7 support. should we have a separate pr instead of mixing in this pr?

If don't remove java7, travis CI would fail.